brackend/tasks/auth.py

```diff
@@ -36,6 +36,8 @@ def wrapper(*args, **kwargs):
             abrt()
 
         g.firebase_id = parsed["uid"]
+        g.user = get_user_by_uid(g.firebase_id)
```

Comment:
I'm thinking we really want to keep any firebase related info as much out of the main application as we can, so when we check ownership/permissions, we should do it on our internal `User.id` instead of using the firebase id, but no real harm having both `user` and `firebase_id` on the global for now

Reply:
This sounds good. We should probably just store the full user object in the globals and access that.